### DISCUSSION TITLE
tailwindcss issue with latest quasar version

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9852] [CLI - PWA mode]
Hello,
I am trying to use the tailwindcss in the quasar project and I m using this extension check [here](https://www.npmjs.com/package/quasar-app-extension-tailwindcss).
So in this extension firstly I was trying to install the tailwindcss@^1.0.0 but it is not working as the quasar is not compatible.
After these, I tried to install the tailwindcss@^2.0.0 but after installing getting issues shown below
![image](https://user-images.githubusercontent.com/57760543/123728940-c58b8c00-d8b1-11eb-8c95-2d796c39d826.png)

So anyone pls help me with the installation of tailwindcss in the quasar project.
@rstoenescu @ontwikkelfabriek can u pls help

**Comment by metalsadman** [comment]
issue a ticket on the maintainer's repo. the extension maybe needed to be updated for QV2.
